.gitignore

```diff
@@ -22,7 +22,7 @@
 
 # Ignore master key for decrypting credentials and more.
 /config/master.key
-
+.env
```

Comment:
Good that you have added to .gitignore but you did it too late because it is already repo.

btw we should have two .env files. One for developemnt and one for test.

We should have two files in repo: `.env.development.example` and `env.test.example` for each development.

Each developer should create copies of them on local machin with local settings:
`.env.development.example` => `.env.development`
`.env.test.example` => `env.test`

`.env.development` and `.env.test` should be placed in .gitignore

Reply:
I did what u've asked. Please check if it's correct